node/alphanet/src/lib.rs

```diff
@@ -229,21 +237,53 @@ mod tests {
             "127.0.0.1:3003",
             3033,
             vec![alice.node_info.clone()],
+            chain_spec.clone(),
+        );
+        let charlie = Node::new(
+            "t2_charlie",
+            "charlie.near",
+            3,
+            "127.0.0.1:3004",
+            3034,
+            vec![bob.node_info.clone()],
             chain_spec,
         );
 
-        let payload = ChainPayload { transactions: vec![], receipts: vec![] };
-        let (beacon_block, shard_block) = match alice.client.try_produce_block(1, payload) {
-            BlockProductionResult::Success(beacon_block, shard_block) => {
-                (beacon_block, shard_block)
-            }
-            _ => panic!("Should produce block"),
-        };
+        let (beacon_block, shard_block) =
+            match alice.client.try_produce_block(1, ChainPayload::default()) {
```

Comment:
To producing a block we also need to add evidence that consensus was achieved in this block (and in the state).